system-addon/lib/TopSitesFeed.jsm

```diff
@@ -128,23 +128,20 @@ this.TopSitesFeed = class TopSitesFeed {
 
   /**
    * Refresh the top sites data for content
-   *
-   * @param target Optional port/channel to receive the update. If not provided,
-   *               the update will be broadcasted.
    */
-  async refresh(target = null) {
+  async refresh(broadcast = true) {
```

Comment:
It's a bit confusing that this is default `true`, but `HighlightsFeed` has it default `false` although it looks like all `fetchHighlights` calls are explicitly passing in a value instead of relying on defaults. Maybe we should just remove the defaults from both places.

Reply:
I suppose if we're touching the callers, might as well clean up the API to not pass in a plain boolean arg, so something more like:
`refresh(false)` -> `refresh({broadcast: false})`
("refresh... not?" vs "refresh without broadcast")